lib/Runtime/Language/i386/AsmJsJitTemplate.cpp

```diff
@@ -1507,6 +1507,16 @@ namespace Js
             int32 stackSize = asmInfo->GetTotalSizeinBytes();
             stackSize = ::Math::Align<int32>(stackSize, 8);
 
+#if DBG_DUMP
+            if (Js::Configuration::Global.flags.DebugBreak.Contains(funcBody->GetFunctionNumber()) ||
+                PHASE_ON(DebugBreakPhase, funcBody))
+            {
+                // int 3
+                *buffer++ = 0xCC;
```

Comment:
It is debug only code.  No need to encode, plus it isn't user controllable.

---
In reply to: [114687392](https://github.com/Microsoft/ChakraCore/pull/2909#discussion_r114687392) [](ancestors = 114687392)